### DISCUSSION TITLE
Show more insights about active community members

### DISCUSSION THREAD

**Discussion by markseuffert** [community/community#138186] [Discussions]
### Select Topic Area

Product Feedback

### Body

The discussions page for your repository shows the `Most helpful` users of the last 30 days. 

Something similar could be useful in the [insights page for your repository](https://docs.github.com/en/discussions/managing-discussions-for-your-community/viewing-insights-for-your-discussions). There are already graphs with `Discussions page views` and `Discussions daily contributors`. What's missing in my opinion is a list of active community members. People who write comments, suggest answers, investigate problems and have friendly conversations with new users. These types of activities save maintainers a lot of time. If you maintain a GitHub repository, you could use this data to get a better understanding of who's contributing to your community. You could also select different periods of time, for example to see the most helpful users of the last year, not just the last 30 days.

Perhaps show a list of active community members? Here's an example:

![Screenshot of two community members](https://github.com/user-attachments/assets/f15d0b19-5a96-4a1b-a677-b4deee029cbe)

**Comment by evilaliv3** [comment]
I consider this proposal very important.

An aspect that i find really to be missing is the possibility to give credits to users that interact with the repository in a whay that is different from committing.

As a project lead, i would find very valuable to to be able to give visibility to user that:

    - created tickets (to value proposals or in general interest into the project)
    - provided answers on discussions
    - get positive community feedback (e.g with emojis reactions)
    - perform review suggestions and in general comments to code

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐
